C/ds/infix_to_postfix.md

```diff
@@ -0,0 +1,19 @@
+# Conversion of an infix expression to postfix expression.
+
+An Infix expression is an expression where operators are written in-between the operands. Example:- A+B.
+Infix expressions are readable and solvable by humans. 
```

Comment:
delete this file and place this content in a comment bottom of your source code

Reply:
Hey this is C file and its already merged to main repo. The one requested to review is C++ file.

Reply:
if it is merged why I am seeing that file here 
delete this file 
edit readme of c++ folder

Reply:
@plazzy99 i have done the changes as you had suggested me. Please do review and let me know if any corrections.